components/financial_assistance/app/assets/javascripts/financial_assistance/step_conditionals.js

```diff
@@ -531,31 +531,40 @@ $(document).ready(function(){
 
   /* Preference Application Form Related */
 
-  /* enable or disable submit application button by electronic siganture (first/last name match)*/
-
-  function enable_submit_button_by_electronic_signature() {
+  /* enable or disable submit application button by checkboxes & electronic siganture (first/last name match)*/
+  function enable_submit_button() {
     first_name_thank_you = $("#first_name_thank_you").val() ? $("#first_name_thank_you").val().toString().toLowerCase().trim() : '';
     last_name_thank_you = $("#last_name_thank_you").val() ? $("#last_name_thank_you").val().toString().toLowerCase().trim() : '';
     subscriber_first_name = $("#subscriber_first_name").val();
     subscriber_last_name = $("#subscriber_last_name").val();
-
-    if(last_name_thank_you == ""){
-      $('.interaction-click-control-submit-application').addClass('disabled');
-    }
-    if( first_name_thank_you == subscriber_first_name && last_name_thank_you == subscriber_last_name){
+    living_outside_no = $('#living_outside_no').is(':checked');
```

Comment:
@j1joey JS variables have the term var, let, or const before them. So you could do `var living_outside_no  =  $('#living_outside_no').is(':checked');`